<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>837</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>841</width>
      <height>381</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QToolButton" name="newFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Users/dvlas/OneDrive/Изображения/Figures/plus.png</normaloff>../../../Users/dvlas/OneDrive/Изображения/Figures/plus.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="openFile">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Users/dvlas/OneDrive/Изображения/Figures/open.png</normaloff>../../../Users/dvlas/OneDrive/Изображения/Figures/open.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="saveFile">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Users/dvlas/OneDrive/Изображения/Figures/save.png</normaloff>../../../Users/dvlas/OneDrive/Изображения/Figures/save.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="colorChange">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Black</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pink</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yellow</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Orange</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Red</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Green</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Blue</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sizeChange">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>18</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>22</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>24</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Size</string>
    </property>
   </widget>
   <widget class="QToolButton" name="search">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Users/dvlas/OneDrive/Изображения/Figures/find.png</normaloff>../../../Users/dvlas/OneDrive/Изображения/Figures/find.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Users/dvlas/OneDrive/Изображения/Figures/clear.png</normaloff>../../../Users/dvlas/OneDrive/Изображения/Figures/clear.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="insertTable">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>0</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>InsTab</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Users/dvlas/OneDrive/Изображения/Figures/insTab.png</normaloff>../../../Users/dvlas/OneDrive/Изображения/Figures/insTab.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="addRow">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Users/dvlas/OneDrive/Изображения/Figures/addrow.png</normaloff>../../../Users/dvlas/OneDrive/Изображения/Figures/addrow.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="rmRow">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>0</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Users/dvlas/OneDrive/Изображения/Figures/rmrow.png</normaloff>../../../Users/dvlas/OneDrive/Изображения/Figures/rmrow.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="addColumn">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>0</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Users/dvlas/OneDrive/Изображения/Figures/rmclm.png</normaloff>../../../Users/dvlas/OneDrive/Изображения/Figures/rmclm.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="rmClm">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>0</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Users/dvlas/OneDrive/Изображения/Figures/addclm.png</normaloff>../../../Users/dvlas/OneDrive/Изображения/Figures/addclm.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>37</width>
      <height>37</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="undo">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Undo</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Users/dvlas/OneDrive/Изображения/Figures/undo.png</normaloff>../../../Users/dvlas/OneDrive/Изображения/Figures/undo.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="copy">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>0</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Users/dvlas/OneDrive/Изображения/Figures/copy.png</normaloff>../../../Users/dvlas/OneDrive/Изображения/Figures/copy.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="pasteText">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Paste</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Users/dvlas/OneDrive/Изображения/Figures/paste.png</normaloff>../../../Users/dvlas/OneDrive/Изображения/Figures/paste.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>430</y>
      <width>841</width>
      <height>341</height>
     </rect>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
